### PR TITLE
GIX-1070: Sns Transfer from wallet page

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

User can make sns transactions from the wallet page.

# Changes

* Add "New Transaction" button to SnsWallet page.
* Add selectedAccount prop to SnsTransactionModal.

# Tests

* Add test for new functionality in SnsWallet page.
* Add test for new prop in SnsTransactionModal

### CODE REVIEW

**Comment by lmuntaner** [comment]
New UX:

https://user-images.githubusercontent.com/4550653/194902343-6c1e41ea-a374-4193-baa8-74a189eaa42b.mov

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

File: frontend/src/lib/pages/SnsWallet.svelte [hunk 1]
```diff
@@ -76,3 +82,22 @@
     {/if}
   </section>
 </main>
+
+<Footer>
+  <Toolbar>
```
Comment (peterpeterparker) [inline frontend/src/lib/pages/SnsWallet.svelte:87]: both `Footer` and `Toolbar` will have to be removed for a `<div class="toolbar">` (I think) once https://github.com/dfinity/nns-dapp/pull/1426 merged

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**